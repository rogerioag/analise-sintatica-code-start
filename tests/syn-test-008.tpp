
inteiro soma(inteiro: a, inteiro: b)
  soma := a + * b  {Erro de sintaxe: operador '*' usado sem operandos.}
  retorna(soma)
fim
