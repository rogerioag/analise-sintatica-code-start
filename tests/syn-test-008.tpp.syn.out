Problema na linha 3, coluna 15, próximo ao token '*'
Erro Sintático: Estado irrecuperável.
Aviso: Não foi possível gerar a Árvore Sintática.
